includes/class-internal-events.php

```diff
@@ -132,30 +145,47 @@ public function force_publish_missed_schedules() {
 	public function confirm_scheduled_posts() {
 		global $wpdb;
 
-		$future_posts = $wpdb->get_results( $wpdb->prepare( "SELECT ID, post_date FROM {$wpdb->posts} WHERE post_status = 'future' AND post_date > %s LIMIT 25;", current_time( 'mysql', false ) ) );
+		$page     = 1;
+		$quantity = 25;
+
+		do {
```

Comment:
I'll bump the limit to 100, but I want to keep this loop. We have a few clients who schedule weeks worth of posts, and get close to the cap of 100. I'd rather confirm everything than risk missing something. This only runs every ten minutes, so I'm not that concerned about the load it'll introduce or the time it'll take to process.